server/src/controllers/onNewTask.ts

```diff
@@ -7,10 +7,10 @@ interface NewTask {
 }
 
 const onNewTask = (io: socketio.Server, socket: socketio.Socket) => ({ roomId, task }: NewTask) => {
-  const room = rooms.getRoom(roomId);
+  const test = parseInt(roomId, 10);
+  const room = rooms.getRoom(test);
```

Comment:
Tak jak rozmawialismy, narazie to moze zostac ale backend powinien przepisac swoj kod by roomId byl stringiem